tests/command/cursor-tailable-001.phpt

```diff
@@ -1,5 +1,5 @@
 --TEST--
-MongoDB\Driver\Cursor tailable iteration with awaitData and maxAwaitTimeMS options
+MongoDB\Driver\Command tailable cursor iteration with maxAwaitTimeMS option
```

Comment:
Also changing the title as originally asked in https://github.com/mongodb/mongo-php-driver/pull/673/files#r153222643